### ISSUE TITLE
File disapperead after download

### ISSUE THREAD

**Issue by gprn** [StefanLobbenmeier/youtube-dl-gui#85]
**Describe the bug**
Video is well downloaded but finally not in Download folder

**To Reproduce**
Download a video and try to find it after download is done

**Additional info (please complete the following information):**
 - OS : Kubuntu 23.04
 - Application version : 2.5.4 but using this app without problem since a lot of version
 - Application type : Appimage downloaded here

**Additional context**
For example, the file I want is downloaded inside a folder in my Download folder called (for the last file) [52c9ab8b6e8dd6d3a7b877b8ae6ab3d7]. I see in live the files well downloaded (.mp4, .mkv, .mp3) but when it's finished, the files disapperead  from this random folder, and nothing is found in the Download folder itself as expected (and as it was before). I think all was fine 1 week before.

**Comment by StefanLobbenmeier** [comment]
Can you check if “prevent saving duplicate filename” is on? There was a bug in that feature that sounds similar

**Comment by gprn** [comment]
Yes, thanks for the advice, all is fine without this option checked
